OneCommunityGlobal/HighestGoodNetworkApp
# Anirudh - Unit test for delete badge pop up

# Description

Unit tests for src/components/Badge/DeleteBadgePopup


## Main changes explained:

    New file
        DeleteBadgePopup.test.js : unit test cases

Test Case

<img width="977" alt="Unit test - DeleteBadgePopup - Test suit" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/47104198/321994e1-a004-421f-821a-94a88a78da04">


…

## How to test:

    check into current branch
    do npm test DeleteBadgePopup.test.js
    check if all the test cases pass


## Screenshots or videos of changes:

<img width="977" alt="Unit test - DeleteBadgePopup" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/47104198/b68e2a68-26c6-41c7-a9ab-8125e77c42e2">


## Note:
Kindly take a look at the test cases and suggest if more validations can be added
